### DISCUSSION TITLE
Developing on Windows, running in docker-linux

### DISCUSSION THREAD

**Discussion by cilerler** [ClearFoundry/ClearScript#235] [Q&A]
How should I target the libraries if I'm developing the app with Windows but deploying it to docker-linux?  Right now, I added both libraries as below, but not sure if it is going to work or not. Should I remove the default one and use the Linux one with docker?  Or the way I already added is the way it should be?

```xml
    <PackageReference Include="Microsoft.ClearScript" Version="7.1.0" />
    <PackageReference Include="Microsoft.ClearScript.linux-x64" Version="7.1.0" />
```

**Comment by ClearScriptLib** [comment]
Hi @cilerler,

That should work. You could probably streamline things slightly by using component packages – e.g., `Microsoft.ClearScript` includes Windows Script engine support that you probably don't need – but what you have should be OK.

Cheers!

**Reply by cilerler** [reply]
Thank you for your prompt reply as usual. 
If I understood what you are saying correctly, the following way should be a more accurate approach? Do you concur?

```xml
    <PackageReference Include="Microsoft.ClearScript.linux-x64" Version="7.1.0" />
    <PackageReference Include="Microsoft.ClearScript.V8.Native.win-x64" Version="7.1.0" />
```

**Reply by ClearScriptLib** [reply]
Yes, that’s probably the most efficient setup for ClearScript V8 on Windows and Linux x64.
